File: 03 C C++ Python R and Mojo review/03-3 R Language Review/04 Macros & Global Vars/01.R
PI <- 3.14159

# Function to calculate area
area <- function(r) {
  return(PI * r * r)
}

# Set radius with conditional logic
radius <- 7  # Default value

if (radius > 10) {
  radius <- 10
} else if (radius < 5) {
  radius <- 5
} else {
  radius <- 7
}

# Calculate and print the area
cat(sprintf("Area of circle with radius %d: %f\n", radius, area(radius)))

File: 03 C C++ Python R and Mojo review/03-2 Python Review/04 Macros & Global Vars/01.py
PI = 3.14159

# Function to calculate the area
def area(r):
    return PI * r * r

# Set radius with conditional logic
radius = 7  # Default value

if radius > 10:
    radius = 10
elif radius < 5:
    radius = 5
else:
    radius = 7

# Calculate and print the area
print(f"Area of circle with radius {radius}: {area(radius)}")